tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
This file has been generated by angular-cli. Most likely `enableIvy` should continue to be `false` because of [here](https://angular.io/guide/ivy#maintaining-library-compatibility). @earshinov what do you think? Should we change the value to `true`?

Reply:
You are right about using `false`. It needs to be `false` to preserve backward compatibility of angular-archwizard library with applications still using ViewEngine.